hackathon_site/dashboard/frontend/src/setupTests.js

```diff
@@ -3,3 +3,14 @@
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
 import "@testing-library/jest-dom/extend-expect";
+
+// Fixes the error I get: "matchMedia not present, legacy browsers require a polyfill"
+window.matchMedia =
+    window.matchMedia ||
+    function () {
+        return {
+            matches: false,
+            addListener: function () {},
+            removeListener: function () {},
+        };
+    };
```

Comment:
Is this the test you use if any images are included ??

Reply:
I added this to fix the error "matchMedia not present, legacy browsers require a polyfill" I was getting when I was trying to run my test. It might be because my yarn version is too ahead of yall. I'll downgrade and try again.

Reply:
ohh okay thanks